web/index.py

```diff
@@ -145,10 +145,10 @@ def filter_logic(self, filters, skip, limit=None):
     cve   = db.getCVEs(limit=limit, skip=skip, query=query)
     # marking relevant records
     if current_user.is_authenticated():
-        if filters['whitelistSelect'] == "on":   cve = self.list_mark('white', cve)
-        if filters['blacklistSelect'] == "mark": cve = self.list_mark('black', cve)
-    self.plugManager.mark(cve, **self.pluginArgs)
-    cve = list(cve)
+        if filters['whitelistSelect'] == "on":   cve['cves'] = self.list_mark('white', cve['cves'])
+        if filters['blacklistSelect'] == "mark": cve['cves'] = self.list_mark('black', cve['cves'])
+    self.plugManager.mark(cve['cves'], **self.pluginArgs)
+   # cve = list(cve)
```

Comment:
Consider removing the comment, the code will not be executed.

Reply:
removed, thanks